pytket/binders/pauli.cpp

```diff
@@ -237,6 +239,185 @@ PYBIND11_MODULE(pauli, m) {
           "The list of Pauli terms")
       .def("__eq__", &PauliStabiliser::operator==)
       .def("__ne__", &PauliStabiliser::operator!=);
+
+  py::class_<QubitPauliTensor>(
+      m, "QubitPauliTensor",
+      "A tensor formed by Pauli terms, consisting of a sparse map from "
+      ":py:class:`Qubit` to :py:class:`Pauli` (implemented as a "
+      ":py:class:`QubitPauliString`) and a complex coefficient.")
```

Comment:
Oh also maybe it would be helpful to update the changelog to inform users that they can now access the `QubitPauliTensor` from python. What do you think?

Reply:
Agreed. I have added a short line to the changelog